src/devices/Mfrc522/Mfrc522Controller.cs

```diff
@@ -0,0 +1,447 @@
+using System;
+using System.Diagnostics;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Collections.Generic;
+
+using static Iot.Device.Mfrc522.Status;
+using static Iot.Device.Mfrc522.Register;
+using static Iot.Device.Mfrc522.RequestMode;
+using static Iot.Device.Mfrc522.Command;
+
+namespace Iot.Device.Mfrc522
+{
+    public class Mfrc522Controller : IDisposable
+    {
+        const byte NRSTPD = 22;
+
+        const byte MAX_LEN = 16;
+
+        private readonly SpiDevice _spiDevice;
+        private GpioController _controller;
+
+        public Mfrc522Controller(SpiDevice spiDevice)
+        {
+            _spiDevice = spiDevice;
+
+            _controller = new GpioController();
+
+            _controller.OpenPin(NRSTPD, PinMode.Output);
+            _controller.Write(NRSTPD, PinValue.High);
```

Comment:
this is set to high twice in a row (here and Init)

Reply:
Yes. I’m aware of that. Planned to be fixed.